web-registry/src/components/organisms/BuyCreditsModal/BuyCreditsModal.utils.ts

```diff
@@ -46,3 +47,23 @@ export const handleBuyCreditsSubmit = async (
   };
   await onSubmit(fullValues);
 };
+
+export const getCreditCountValidation =
+  (creditAvailable: number) => (creditCount: number) => {
+    let error;
+    if (creditCount > creditAvailable) {
+      error = `The amount of credits must be lower or equal than the max credit(s) available (${creditAvailable}).`;
+    }
+    return error;
+  };
+
+type AmountToSpendParams = {
+  creditCount: number;
+  askAmount: number;
+};
+
+export const amountToSpend = ({
+  creditCount,
+  askAmount,
+}: AmountToSpendParams): String =>
+  formatNumber(creditCount * microToDenom(askAmount), 2) ?? '-';
```

Comment:
Trim the amount to spend decimals to 2, before we could have number like this:
![image](https://user-images.githubusercontent.com/530644/187913116-29dd19ae-0669-4b64-9278-7b91203e0866.png)